### PR TITLE
[*] TEST : speed up php parse error checker & precompute travis composer.lock

### PR DESCRIPTION
**Pull request by jocel1**

### CODE REVIEW

**Comment by jocel1** [comment]
still WIP

File: tests/composer.json.travis [hunk 1]
```diff
@@ -0,0 +1,170 @@
+{
+  "name": "PrestaShop/PrestaShop",
+  "description": "PrestaShop offers a free, fully scalable, Open Source e-commerce solution.",
+  "type": "project",
+  "repositories": [
+    {"type": "vcs", "url": "https://github.com/prestashop/blockcart"},
```
Comment (maximebiloe) [inline tests/composer.json.travis]: Please, only use `composer` to download the modules so the tests will truly reflect the real package

Comment (mickaelandrieu) [inline tests/composer.json.travis]: @maximebiloe it's [composer](https://getcomposer.org/doc/05-repositories.md#vcs) :)

Comment (maximebiloe) [inline tests/composer.json.travis]: Yes, I meant to only use the packagist repos set below. I used the wrong word, my bad :wink:

Comment (jocel1) [inline tests/composer.json.travis]: actually the composer.json.travis is just a copy of the existing composer.json :)

Comment (maximebiloe) [inline tests/composer.json.travis]: Yes, I just saw that. I've removed some lines a long time ago and didn't remember there was still much :/ 
Sorry

**Comment by mickaelandrieu** [comment]
Thank you @jocel1

**Pull request merged**